packages/zwave-js/src/lib/node/Node.ts

```diff
@@ -636,9 +636,15 @@ export class ZWaveNode extends ZWaveNodeMixins implements QuerySecurityClasses {
 
 				const shouldUpdateOptimistically =
 					api.isSetValueOptimistic(valueId)
+					// Check if the device class supports optimistic value updates
+					&& (endpointInstance.deviceClass?.specific
+						?.supportsOptimisticValueUpdate
+						?? endpointInstance.deviceClass?.generic
+							?.supportsOptimisticValueUpdate
+						?? true)
```

Comment:
This seems redundant. If `deviceClass` is defined, so is `deviceClass.specific` and its `supportsOptimisticValueUpdate` property has also been initialized.


```suggestion
					&& (endpointInstance.deviceClass?.specific.supportsOptimisticValueUpdate
						?? true)
```